server/src/server.ts

```diff
@@ -40,8 +44,11 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 export function start(port: number) {
   app.listen(port ? port : 7777);
-}
 
-if (process.argv[2]) {
-  start(+process.argv[2]);
+  // install plugins
+  console.log('starting----');
```

Comment:
Do you mean to have this duplicated 4 times?

Reply:
Removed it